src/selection/highlight/highlight.js

```diff
@@ -94,8 +94,12 @@ class Highlight {
    * @returns {boolean}
    */
   isEnabledFor(highlightType, coords) {
+    let type = highlightType;
```

Comment:
Thanks for the question 👍 We can comment styles. Especially, when in our opinion there is more clear or efficient way to write something. However, it may be quite subjective. The Golden mean is quite hard to determine.

For me, using `let` here is much more clear compared to ternary operator (by the way, I don't like it) mixed with value assignment.

In such situation we usually accept solution provided by developer creating PR. For quite important things we ask team about their opinion.